functions/src/index.ts

```diff
@@ -143,3 +145,124 @@ export const beforecreated: BlockingFunction = beforeUserCreated(
     }
   },
 )
+
+export const exportHealthSummary = onRequest(async (req, res) => {
```

Comment:
As noted above; we should aim to use v2 of cloud functions and use `onCall` here.

A task for the future: We should also aim to attach a specific service account to the function execution and need to make it "public" + require authentication in the future. We should take some inspiration from the TDS approved setup in https://github.com/StanfordBDHG/PediatricAppleWatchStudy

Reply:
Yup, already on it. I will have a look at the other repo as well.

Reply:
Sounds great; that would be perfect!

Reply:
This is non-blocking for the merge, right? Because the adaption regarding the onRequest > onCall would be planned for a future PR rather than a new commit on this branch.

Reply:
Sure; would be fine for me. Adding a test here would be great and we should do that for future PRs but I am fine if we add a test in a later PR as and make this an exception for now.